### PR TITLE
[14.0][FW] stock_split_picking: Port PR from 13.0 to 14.0

### PR DESCRIPTION
**Pull request by sonhd91**
This PR migtate [PR#858](https://github.com/OCA/stock-logistics-workflow/pull/858) to 14.0

### CODE REVIEW

File: stock_split_picking/README.rst [hunk 1]
```diff
@@ -52,6 +52,9 @@ Usage
 Bug Tracker
 ===========
 
+* When splitting a picking in an unassigned state, wizard won't
+  be auto completed with picking lines.
+
```
Comment (sebalix) [inline stock_split_picking/README.rst]: These changes will be overridden automatically by a bot as the `README.rst` file is automatically generated.
It's ok to remove them, I don't think we need to document this.

Comment (rousseldenis) [reply stock_split_picking/README.rst]: @sonhd91 Indeed, please put this in a supported section.

See https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Comment (sonhd91) [reply stock_split_picking/README.rst]: I will update it in USAGE.rst file and auto generate README again

**Review by leemannd — APPROVED** [review]

**Review by sebalix — APPROVED** [review]

**Pull request merged**